src/searchSite.ts

```diff
@@ -21,15 +21,17 @@ const searchSite = async (domain: string, searchTerm: string) : Promise<URL[]> =
     page.keyboard.press('Enter'),
   ]);
 
+  // TODO: Navigate all the pages not just first.
   const links = await page.$$eval('a', (as) => as.map((a) => (a as HTMLAnchorElement).href));

```

Comment:
We can scrap all the existing puppeter code in favor of using the library, we don't need to create our own browser object or any of that.